syntax = "proto3";
package message;

service VerifyService {
  rpc GetVerifyCode (GetVerifyReq) returns (GetVerifyRsp) {}
}

enum ErrorCode {
  Succeed = 0;
  Error = 1;
}

message GetVerifyReq {
  string email = 1;
}
message GetVerifyRsp {
  ErrorCode error_code = 1;
  string email = 2;
  string code = 3;
}